ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable-4.17-upgrade-from-stable-4.17.yaml

```diff
@@ -0,0 +1,330 @@
+base_images:
+  ansible:
+    name: "4.17"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.17"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tools:
+    name: "4.17"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      relative: 1
+      version_bounds:
+        lower: 4.17.0-0
+        stream: 4-dev-preview
+        upper: 4.18.0-0
+  target:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.17.0-0
+        stream: 4-dev-preview
+        upper: 4.18.0-0
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aws-c2s-ipi-disc-priv-fips-f28
```

Comment:
I have no idea where you copied these stable -> stable ci jobs, but sounds like it breaks our previous agreement - https://redhat-internal.slack.com/archives/GH9JHQFKN/p1681696518621429

Reply:
Jian Li is probably unable to watch that channel. Copying the message here.

```
we already moved some steps forward to refine stable ci, now only run basic health checks for install and upgrade testing in stable ci, instead of running the whole e2e test suites. Next we will continue to refine configuration test coverage in stable ci, and propose to cover only some configurations which may have some differences in stable builds, e.g:  disconnected, behind proxy, fips. Maybe select ~ 4 configurations on each platform to cover in stable ci. So make sure each lead confirm this proposal and keep us on the agreement.
Keep in mind that stable builds are prompted from nightly builds, we are supposed to find out issues on nightly ci as early as possible, but not rely on stable ci to find out the issues.
```

Reply:
The jobs copied from:
1. ocp 4.16 stable -> stable config
2. the missed e2e profiles from ocp 4.17 stable

Reply:
Get it.

Reply:
If we are going to align the stable -> stable upgrade ci with e2e stable ci, we can use the e2e stable ci jobs override the upgrade ci jobs instead of merging both.

Reply:
This should be new approach because I always copy 4.n-1 stable for 4.n stable, so need confirm with others.

Reply:
Currently 4.16 stable -> stable ci, we only have 4 ci jobs:
````
openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml:- as: aws-ipi-disc-priv-tp-f28
openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml:- as: azure-mag-ipi-fullyprivate-f14
openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml:- as: gcp-ipi-proxy-tp-f28
openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml:- as: nutanix-ipi-proxy-fips-mini-perm-f28
````
for the 2 tp jobs are added on purpose for OCP cases, so only `azure-mag-ipi-fullyprivate` need to confirm, I already left a comment for the job in the following.